cobalt/dom/media_source_attachment.h

```diff
@@ -32,20 +35,19 @@ class MediaSourceAttachment
  public:
   typedef web::UrlRegistry<MediaSourceAttachment> Registry;
 
-  explicit MediaSourceAttachment(scoped_refptr<MediaSource> media_source)
-      : media_source_(media_source) {}
-
-  scoped_refptr<MediaSource> media_source() const { return media_source_; }
+  MediaSourceAttachment() = default;
+  ~MediaSourceAttachment() = default;
 
-  void TraceMembers(script::Tracer* tracer) override {
-    tracer->Trace(media_source_);
-  }
+  virtual bool StartAttachingToMediaElement(
```

Comment:
It seems that there are a lot of comments in the Chromium counterpart (http://shortn/_sXmzZV6yQI).  Consider putting more comments here so it's easier to understand the intention.

Reply:
Done, I used the Chromium comments as a base, then removed the portions of the comments that are unrelated to Cobalt.